app/javascript/packs/stylesheets/bootstrap.scss

```diff
@@ -250,6 +250,13 @@ select.form-control {
   z-index: 1999;
 }
 
+.laboratory-modal-container {
```

Comment:
this can be combined with the two above it (saves space):
```
.confirm-export-modal-container, .advanced-filter-modal-container, .laboratory-modal-container {
...
}
```

Reply:
https://github.com/SaraAlert/SaraAlert/pull/861/commits/0b00f9efd83684aa21526ddee93b433c20c8fb79#diff-08fe027fbe0e1a0481ee292a8b7590c2b3aeeab2c63aeeab1dabb039918e9357R248